app/controllers/questions_controller.rb

```diff
@@ -57,129 +11,6 @@ def lottery
-    end
-
-    updates = {
-      post: @question,
-      user: current_user,
-      community: @question.community,
-      body: body_rendered,
-      title: params[:question][:title] == @question.title ? nil : params[:question][:title],
-      tags_cache: new_tags_cache == @question.tags_cache ? @question.tags_cache : new_tags_cache,
-      body_markdown: body_markdown,
-      comment: params[:edit_comment],
-      active: true, accepted: false,
-      decided_at: nil, decided_by: nil,
-      rejected_comment: nil
-    }
-
-    @edit = SuggestedEdit.new(updates)
-    if @edit.save
-      @question.user.create_notification("Edit suggested on your post #{@question.title.truncate(50)}",
-                                         question_url(@question))
-      redirect_to share_question_path(@question)
-    else
-      @question.errors = @edit.errors
-      render :edit
-    end
-  end
-
-  def destroy
-    unless check_your_privilege('flag_curate', @question, false)
-      flash[:danger] = helpers.ability_err_msg(:flag_curate, 'delete this question')
-      redirect_to(question_path(@question)) && return
-    end
-
-    if @question.answer_count.positive? && @question.answers.any? { |a| a.score >= 0.5 }
-      flash[:danger] = 'This question cannot be deleted because it has answers.'
-      redirect_to(question_path(@question)) && return
-    end
-    if @question.deleted
-      flash[:danger] = "Can't delete a deleted question."
-      redirect_to(question_path(@question)) && return
-    end
-
-    if @question.update(deleted: true, deleted_at: DateTime.now, deleted_by: current_user,
-                        last_activity: DateTime.now, last_activity_by: current_user)
-      PostHistory.post_deleted(@question, current_user)
-    else
-      flash[:danger] = "Can't delete this question right now. Try again later."
-    end
-    redirect_to url_for(controller: :questions, action: :show, id: @question.id)
-  end
-
-  def undelete
-    unless check_your_privilege('flag_curate', @question, false)
-      flash[:danger] = helpers.ability_err_msg(:flag_curate, 'undelete this question')
-      redirect_to(question_path(@question)) && return
-    end
-
-    unless @question.deleted
-      flash[:danger] = "Can't undelete an undeleted question."
-      redirect_to(question_path(@question)) && return
-    end
-
-    if @question.deleted_by.is_moderator && !current_user.is_moderator
-      flash[:danger] = 'You cannot undelete this post deleted by a moderator.'
-      redirect_to(question_path(@question)) && return
-    end
-
-    if @question.update(deleted: false, deleted_at: nil, deleted_by: nil,
-                        last_activity: DateTime.now, last_activity_by: current_user)
-      PostHistory.post_undeleted(@question, current_user)
-    else
-      flash[:danger] = "Can't undelete this question right now. Try again later."
-    end
-    redirect_to url_for(controller: :questions, action: :show, id: @question.id)
-  end
-
   def feed
```

Comment:
Question: Is there a specific reason to have (a) this method in Questions controller and (b) not for Articles (controller was deleted)

Reply:
This is where the RSS feed is sourced from. RSS feeds aren't something I've touched in this PR, so it's just left as-is. May need reviewing in the future to see if it still does what we want it to.